### PR TITLE
Change titles of re-branding docs

### PR DESCRIPTION
**Pull request by sherenewso2**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -595,12 +595,11 @@ nav:
...
             - 'Entitlement with APIs': develop/entitlement-with-apis.md
-      - 'Customizing User Interfaces':
-        - 'Customizing WSO2 Identity Server UIs': develop/customizing-identity-server-uis.md
-        - 'Customizing Login Pages for Service Providers': develop/customizing-login-pages-for-service-providers.md
-        - 'Customizing the SSO Redirection Page': develop/customizing-the-sso-redirection-page.md
-        - 'Customizing the User Portal UI': develop/customizing-user-portal-ui.md
-      - 'Customizing the Authentication Endpoint': develop/customizing-the-authentication-endpoint.md
+      - 'Re-branding User Interfaces':
+        - 'Re-branding WSO2 Identity Server UIs': develop/customizing-identity-server-uis.md
+        - 'Re-branding the Default Login Page for Your Application ': develop/customizing-login-pages-for-service-providers.md
```
Comment (omindu) [inline en/mkdocs.yml:600]:
```suggestion
        - 'Re-branding the Default Login Page for Your Application': develop/customizing-login-pages-for-service-providers.md
```

File: en/docs/develop/customizing-identity-server-uis.md [hunk 2]
```diff
@@ -16,15 +17,14 @@ All these UIs are available in the **`authenticationendpoint`** and **`accountre
...
-        You will be adding the customized JSP files along with the custom CSS files into this new `extensions` folder.
+        You will be adding the modified JSP files along with the modified CSS files into this new `extensions` folder.
            
 2. Similarly, navigate to the `<IS_HOME>/repository/deployment/server/webapps/accountrecoveryendpoint` folder and create an `extensions` folder.
 
 
 ## Step 2: Copy the existing header and footer content
 
-In order to customize the header and footer content of the web applications, first you need to copy them to the newly cerated `extensions` folders. 
+To re-brand the header and footer content of the web applications, first copy them over to the newly created `extensions` folders.
```
Comment (gomathyk) [inline en/docs/develop/customizing-identity-server-uis.md:27]: `extensions` folder.

File: en/docs/develop/customizing-login-pages-for-service-providers.md [hunk 3]
```diff
@@ -51,21 +41,18 @@ customized one.
...
         
             #openid.return_to parameter
             OpenId.ReturnToURL=http://wso2.is:8080/avis.com/home.jsp
             ```
-3. Register a new service provider with name `avis.com` similarly as
-   explained in [configuring service provider for travelocity
-                                                sample](../../deploying-the-sample-app/#configuring-the-service-provider)
-   while replacing `travelocity.com` references to `avis.com`.
+
+4. Register a new service provider with the name `avis.com` similarly by following the steps given in the [configuring service provider for travelocity sample section](../../deploying-the-sample-app/#configuring-the-service-provider) while replacing `travelocity.com` references to `avis.com`.
```
Comment (gomathyk) [inline en/docs/develop/customizing-login-pages-for-service-providers.md:53]: similarly,

File: en/docs/develop/customizing-login-pages-for-service-providers.md [hunk 4]
```diff
@@ -123,34 +110,34 @@ customized one.
...
 ```
 [authentication.endpoints] 
 login_url="/authenticationendpoint/login.do"
 retry_url="/authenticationendpoint/retry.do" 
 ```
 
-By default it points to a location inside the Identity Server itself,
-thus the relative path is given. If it is necessary to point to an
+By default, the configuration points to a location inside WSO2 Identity Server itself
+thus, the relative path is given. If it is necessary to point to an
```
Comment (gomathyk) [inline en/docs/develop/customizing-login-pages-for-service-providers.md:135]: itself,

File: en/docs/develop/customizing-login-pages-for-service-providers.md [hunk 5]
```diff
@@ -123,34 +110,34 @@ customized one.
...
 external application, the full path should be given instead.
 
 !!! note
     
-    If this web app is moved outside the Identity Server, we must ensure
+    If this web app is moved outside the Identity Server, ensure
```
Comment (gomathyk) [inline en/docs/develop/customizing-login-pages-for-service-providers.md:140]: outside WSO2 Identity Server,

File: en/docs/develop/customizing-login-pages-for-service-providers.md [hunk 6]
```diff
@@ -182,32 +169,30 @@ forward the request to the url pattern
 **login.jsp** file. The request is finally forwarded to this
 **login.jsp** page.
 
-Everything on the authententicationendpoint web application is
-customizable. You can customize it by adding J SP pages or modifying
-them and configuring the **web.xml** respectively.
+Everything on the **authenticationendpoint** web application can be modified according to your requirements. You can either add JSP pages or modify them and configure the **web.xml** respectively.
```
Comment (gomathyk) [inline en/docs/develop/customizing-login-pages-for-service-providers.md:172]: **web.xml** file

**Pull request merged**